packages/connect-explorer/src/pages/changelog.mdx

```diff
@@ -2,19 +2,20 @@
 icon: LOG
 ---
 
-import { useData } from 'nextra/data';
-import { serialize } from 'next-mdx-remote/serialize';
-import { MDXRemote } from 'next-mdx-remote';
 import rehypeSectionize from '@hbsnow/rehype-sectionize';
 import remarkGfm from 'remark-gfm';
 import remarkGemoji from 'remark-gemoji';
+import useSWR from 'swr';
+import Markdown from 'react-markdown';
+import { useMDXComponents } from '@trezor/connect-explorer-theme';
+import { Callout } from 'nextra/components';
 
 import { spacings } from '@trezor/theme';
 import { CollapsibleBox } from '@trezor/components';
 
-export const getStaticProps = ({ params }) => {
+export const loadData = () => {
     return fetch(
-        `https://raw.githubusercontent.com/trezor/trezor-suite/${process.env.COMMIT_HASH}/packages/connect/CHANGELOG.md`,
+        `https://raw.githubusercontent.com/trezor/trezor-suite/develop/packages/connect/CHANGELOG.md`,
```

Comment:
The problem behind this PR is that when we create a canary release, the [changelog deployed on prod]( https://connect.trezor.io/9/changelog/) doesn't update. There is still has the old canary release

Reply:
I see, but I still don't like that changing develop changes production. Don't we have some kind of predictable branching pattern that could handle this? What does @karliatto think?

Reply:
We would have to cherrypick the changelog changes from canary into the current prod release branch. Another option would be to pull the current package versions from NPM, not the changelog.

Reply:
Ok, that solves the NPM packages, but then there's still `connect.trezor.io/9`

Reply:
You are right, but in practice it should always be the same as NPM. 

With the current release process it should always be the same latest canary version in NPM `@trezor/connect` as in connect.trezor.io/9.x.x, of course it is possible that some day it fails because we do not complete the release for some reason, but that would some error or similar and we should fix it.

Reply:
So for now I will load the latest version from NPM and from that determine the latest branch where I will get CHANGELOG.md from Github